src/test/java/no/unit/nva/cristin/projects/CommonUtilsTest.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.cristin.projects;
+
+import static no.unit.nva.cristin.projects.CommonUtils.hasValidContent;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.util.HashMap;
+import no.unit.nva.cristin.projects.model.cristin.CristinProject;
+import org.junit.jupiter.api.Test;
+
+public class CommonUtilsTest {
```

Comment:
I don't think there is any reason to have such tests; this is very close to the implementation and doesn't add to coverage unless the methods are untested from an API-unit-test. In the latter case, it would cover up potential issues in the code (the method was in fact untested from the API).